mdal/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ SET(MDAL_SOURCES
   mdal.cpp
   mdal_utils.cpp
   mdal_loader.cpp
+  mdal_defines.cpp
```

Comment:
Maybe mdal_data_model would be a better name... I was expecting a file full of #define statements :-)